easybuild/easyconfigs/p/PCMSolver/PCMSolver-20160205-intel-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'CMakeMake'
+
+name = 'PCMSolver'
+version = '20160205'
+
+homepage = 'https://pcmsolver.readthedocs.org'
+description = """An API for the Polarizable Continuum Model."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'cstd': 'c99'}
+
+sources = ['2473699d6c5db36525160e.tar.gz']
+source_urls = ['https://github.com/PCMSolver/pcmsolver/archive/']
+
+python = 'Python'
+pyver = '2.7.11'
+versionsuffix = '-%s-%s' % (python, pyver)
+
+dependencies = [
+    (python, pyver),
+    ('zlib', '1.2.8'),
+    ('Boost', '1.59.0', versionsuffix),
+]
+
+builddependencies = [
+    ('CMake', '3.4.1'),
```

Comment:
3.4.3, see https://github.com/hpcugent/easybuild-easyconfigs/pull/2401

Shall I flesh it out?